frontend/ui/widget/keyvaluepage.lua

```diff
@@ -130,36 +131,56 @@ function KeyValueItem:init()
         }
     end
 
-    local key_w = RenderText:sizeUtf8Text(0, self.width, self.cface, self.key).x
-    local value_w = RenderText:sizeUtf8Text(0, self.width, self.cface, self.value).x
-    if key_w + value_w > self.width then
-        -- truncate key or value so they fits in one row
-        if key_w >= value_w then
-            self.show_key = RenderText:truncateTextByWidth(self.key, self.cface, self.width-value_w)
-            self.show_value = self.value
+    local frame_padding = 8
```

Comment:
From @poire-z's suggestion to add padding? That is, just from playing around with a few values.